### PR TITLE
Update flattenLayer functions to prepare for new ODC config

### PR DESCRIPTION
**Pull request by wadhwamatic**
This PR allows PRISM to work with a new structure for layers in a hierarchy served from WFP's open data cube (ODC) instance. The changes in ODC required layers to be 'flattened' on the PRISM side when we fetch available layers and dates using a WMS get capabilities request.

For testing purposes, all WMS layers coming from WFP have been updated to a new domain (https://ows.earthobservation.vam.wfp.org). The new ODC configuration will be deployed to odc.ovio.org. Once testing is completed, we will revert back to https://odc.ovio.org and merge this branch

The changes in the WMS responses can be seen by comparing :
https://odc.ovio.org/wms?request=GetCapabilities
https://ows.earthobservation.vam.wfp.org/wms?request=GetCapabilities

### CODE REVIEW

**Comment by wadhwamatic** [comment]
deployed here: https://new-ows-moz.surge.sh/

All is working as expected except for exposure analysis of a tropical storm in which case we get a 500 error: 
error: "500 Internal Server Error: The file you requested is not available - https://ows.earthobservation.vam.wfp.org/?service=WCS&request=GetCoverage&version=2.0.0&coverageId=wp_pop_cicunadj&subset=Long(30.215605, 40.84062576300005)&subset=Lat(-26.867040757999973, -10.472198489999926)"

To replicate go to https://new-ows-moz.surge.sh/?hazardLayerId=adamts_buffers&date=2022-01-24 and click 'exposure analysis'

**Comment by wadhwamatic** [comment]
See issue https://github.com/WFP-VAM/prism-frontend/issues/423

**Comment by wadhwamatic** [comment]
replaced all occurrences of https://odc.ovio.org/ with https://ows.earthobservation.vam.wfp.org/. this is temporary so we can test a new ows config on the latter. once testing is done, revert back to odc.ovio.org before merging

File: src/config/global/prism.json [hunk 1]
```diff
@@ -2,36 +2,120 @@
...
-        "dry_spells": ["streak_dry_days"],
-        "heavy_rain": ["days_heavy_rain", "streak_heavy_rain"]
-      },
-      "vegetation":{
-        "vegetation_conditions": ["ndvi_dekad", "ndvi_dekad_anomaly"]
-      },
-      "temperature": {
-        "land_surface_temperature": ["lst_daytime", "lst_anomaly", "lst_amplitude"]
-      }
+  "categories": {
```
Comment (ericboucher) [inline src/config/global/prism.json:23]: @wadhwamatic are these changes intentional?

**Review by ericboucher — APPROVED** [review]

File: src/config/global/prism.json (hunk 1 above)
Comment (wadhwamatic) [reply src/config/global/prism.json:23]: hmm.. sorry, the global changes should not have been here. let's remove those
